developerDocs/advanced-use-cases.md

```diff
@@ -0,0 +1,183 @@
+---
+title: Advanced Use Cases
+category: 64cbb5277b5f3c0065d96616
+slug: opensea-sdk-advanced-use
+parentDocSlug: opensea-sdk
+order: 2
+hidden: false
+---
+
+- [Scheduling Future Listings](#scheduling-future-listings)
+- [Purchasing Items for Other Users](#purchasing-items-for-other-users)
+- [Using ERC-20 Tokens Instead of Ether](#using-erc-20-tokens-instead-of-ether)
+- [Private Auctions](#private-auctions)
+- [Listening to Events](#listening-to-events)
+
+## Advanced
+
+Interested in purchasing for users server-side or with a bot, scheduling future orders, or making bids in different ERC-20 tokens? OpenSea.js can help with that.
+
+### Scheduling Future Listings
+
+You can create sell orders that aren't fulfillable until a future date. Just pass in a `listingTime` (a UTC timestamp in seconds) to your SDK instance:
+
+```typescript
+const auction = await openseaSDK.createSellOrder({
+  tokenAddress,
+  tokenId,
+  accountAddress,
+  startAmount: 1,
+  listingTime: Math.round(Date.now() / 1000 + 60 * 60 * 24), // One day from now
+});
+```
```

Comment:
i know of some python tooling that does this, but is there any easy integration that tests/lints code blocks in your docs?

might be worth doing (eventually) to keep our docs functional as we add more examples in markdown like this

Reply:
https://github.com/eslint/eslint-plugin-markdown#eslint-plugin-markdown

Reply:
prettier should automatically format code blocks in markdown files as long as it runs on md files, i checked package.json lint-staged rules and it should be running on *.md files

Reply:
sweet, good to know! typechecking and all that out of the box too?